### PR TITLE
feat(stats): add stats endpoint for marketing

### LINKED ISSUE #363: Build a stats endpoint for marketing
Should include:

- [ ] Tips
- [ ] ReTips
- [ ] Unique (Re-)Tippers
- [ ] Unique Commenters
- [ ] Total Amount (Re-)Tipped

...in the last 1, 7, 30 days

### PR DESCRIPTION
**Pull request by thepiwo**
closes #363

### CODE REVIEW

**Review by kenodressel — CHANGES_REQUESTED** [review]
We should ~remove~ move this: https://github.com/aeternity/tipping-community-backend/blob/main/modules/domains/logic/staticLogic.js#L9 as its superseded by your PR. You can also copy the tests from there, they should be mostly applicable to this. Maybe you could also migrate these to the stats module :)

File: modules/stats/routes/statsRoutes.js [hunk 1]
```diff
@@ -50,4 +50,24 @@ router.get('/sender', async (req, res) => {
...
+ *       200:
+ *         description: Returns marketing stats
+ *         content:
+ *           application/json:
+ *             schema:
+ *               type: object
+ */
+router.get('/marketing', async (req, res) => {
+  const stats = await StatsLogic.fetchMarketingStats();
+  return stats ? res.send(stats) : res.sendStatus(404);
```
Comment (kenodressel) [inline modules/stats/routes/statsRoutes.js:70]: maybe just return stats, 404 is not really an a true error here

Comment (thepiwo) [reply modules/stats/routes/statsRoutes.js]: done

File: modules/stats/routes/statsRoutes.js [hunk 2]
```diff
@@ -50,4 +50,24 @@ router.get('/sender', async (req, res) => {
...
+ *         description: Returns marketing stats
+ *         content:
+ *           application/json:
+ *             schema:
+ *               type: object
+ */
+router.get('/marketing', async (req, res) => {
+  const stats = await StatsLogic.fetchMarketingStats();
+  return res.send(stats);
+});
```
Comment (kenodressel) [inline modules/stats/routes/statsRoutes.js:71]:
```suggestion
router.get('/marketing', async (req, res) => res.send(await StatsLogic.fetchMarketingStats()));
```

**Review by kenodressel — APPROVED** [review]

**Pull request merged**